Packs/PhishingURL/ReleaseNotes/1_1_18.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### DBotPredictURLPhishing
+
+- Added the new command `requestProtocol`: The protocol to use when calling the URLs. This argument effects the calls sent by the model only and has no effect on the rasterize or whois commands.
```

Comment:
```suggestion
- Added the new argument `defaultRequestProtocol` (default value http). This will be the default protocol used for URLs supplied with no explicit schema. This argument affects the http requests sent by this script, it has no effect to scripts called through executeCommand (rasterize, whois).
```

Reply:
make sure you comment in the jira about this argument